service/controller/v7/cloudconfig/master_template.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	providerv1alpha1 "github.com/giantswarm/apiextensions/pkg/apis/provider/v1alpha1"
 	"github.com/giantswarm/certs"
-	k8scloudconfig "github.com/giantswarm/k8scloudconfig/v_4_1_2"
```

Comment:
We also seem to have skipped `v_4_2_0` here.

Reply:
As we discussed in Azure we try to make the bump and see how it goes.